.pre-commit-config.yaml

```
@@ -0,0 +1,27 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+ci:
+    autofix_commit_msg: |
+      [pre-commit.ci] auto code formatting
+    autofix_prs: true
```

At least to start, I think we want to avoid any surprises by automatically committing changes to people's PRs. Instead, we can leave a PR comment with `pre-commit.ci autofix` to run. 

```suggestion
    autofix_prs: false
```

---

Great. I was hoping to ask that question!